CMakeLists.txt

```diff
@@ -306,9 +313,11 @@ set(CMAKE_CXX_FLAGS_DEBUG   " -DDEBUG -g3 -ggdb3")
 
 if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
   add_definitions(-DOS_LINUX)
+  SET(BUILD_BLE_SUPPORT OFF)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "QNX")
   add_definitions(-DOS_QNX)
   SET(BUILD_BT_SUPPORT OFF)
+  SET(BUILD_BLE_SUPPORT OFF)
   SET(BUILD_BACKTRACE_SUPPORT OFF)
   SET(EXTENDED_MEDIA_MODE OFF)
 endif()
```

Comment:
@VadymLuchko I would make this flag OFF by default and add here:
```
elseif(ANDROID)
SET(BUILD_BLE_SUPPORT ON)
endif()
```

Reply:
fixed in c946573aa403908e9e337fd1e51e0eb1a8288463